scripts/deploy/deploy-webapi.sh

```diff
@@ -36,6 +38,11 @@ while [[ $# -gt 0 ]]; do
         ;;
         -p|--package)
         PACKAGE_FILE_PATH="$2"
+        shiftregister-app
```

Comment:
is this right??
```suggestion
        shift
```

Reply:
That was a copy/paste error.  Fixed thanks.